packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3191,6 +3191,7 @@ void main() {
 //    properties.add(DiagnosticsProperty<TextStyle>('style', style, defaultValue: null));
 //    properties.add(DiagnosticsProperty<bool>('autofocus', autofocus, defaultValue: false));
 //    properties.add(DiagnosticsProperty<bool>('showCursor', showCursor, defaultValue: false));
+//    properties.add(IntProperty('minLines', minLines, defaultValue: null));
```

Comment:
I think that this comment block was meant to be removed from the unit test. Should we delete the comments instead?

cc/ @chunhtai

Reply:
Yes, I think i accidentally include this comment in this file, please remove these comments

Reply:
alright